eng/Versions.props

```diff
@@ -33,7 +33,7 @@
     <CodeStyleAnalyzerVersion>3.5.0-beta3-20078-04</CodeStyleAnalyzerVersion>
     <VisualStudioEditorPackagesVersion>16.4.248</VisualStudioEditorPackagesVersion>
     <ILToolsPackageVersion>5.0.0-alpha1.19409.1</ILToolsPackageVersion>
-    <MicrosoftVisualStudioLanguageServerPackagesVersion>16.3.27-develop-gdd55e997</MicrosoftVisualStudioLanguageServerPackagesVersion>
+    <MicrosoftVisualStudioLanguageServerPackagesVersion>16.5.1043-develop-gf07d514b</MicrosoftVisualStudioLanguageServerPackagesVersion>
```

Comment:
I believe referencing 16.5 assemblies means this needs to target vs-deps until 16.5 is released.

Reply:
I'll retarget once master-vs-deps is up to date with master.  Leaving as-is so it's reviewable.